README.md

```diff
@@ -72,6 +72,14 @@ There are two ways to install Easy Spaces:
     sfdx force:user:permset:assign -n SpaceManagementApp
     ```
 
+1. (Optional) Assign the `Walkthroughs` permission set to the default user.
```

Comment:
This adds instructions to assign the necessary permission set. This is auto-created in orgs when the walkthroughs feature is enabled.